ota-plus-web/app/com/advancedtelematic/auth/oidc/OidcGateway.scala

```diff
@@ -245,7 +254,8 @@ object OidcConfiguration {
   def load(wSClient: WSClient, configUrl: Url, issuerUrl: Url, log: Logger)(
       implicit ec: ExecutionContext
   ): Future[ProviderMetadata] = {
-    val uri = Uri(configUrl).withPath(OidcConfigurationPath)
+    val root = Uri(configUrl)
+    val uri  = root.withPath(root.path ++ OidcConfigurationPath)
```

Comment:
Is this a bugfix?

Reply:
More like support for additional options from standard, which we need for angel delight